pymc/distributions/continuous.py

```diff
@@ -2561,10 +2567,11 @@ def logcdf(value, nu):
 
         Parameters
         ----------
-        value: numeric or np.ndarray or `TensorVariable`
+        value : tensor_like of float
             Value(s) for which log CDF is calculated. If the log CDF for
             multiple values are desired the values must be provided in a numpy
             array or `TensorVariable`.
```

Comment:
Hi @OriolAbril, should we replace `TensorVariable` in the description with :class:\`aesara.tensor.TensorVariable`

Reply:
I noticed in [pymc.Uniform](https://docs.pymc.io/en/latest/api/distributions/generated/pymc.Uniform.html)
the value is mentioned as `numeric or ndarray or TensorVariable` but in this PR I am using `tensor_like of float` which one is right ?

https://github.com/pymc-devs/pymc/blob/7cc570a867fa8a4f8018d91c0aab1cf2aab64258/pymc/distributions/continuous.py#L319-L322

Reply:
This should be `tensor_like`. I haven't had time to update the guide. While reviewing PRs and while chatting with Ricardo during the 2nd sprint session we realized it would be better to define this `tensor_like` concept in our glossary instead of using numeric, ndarray or TensorVariable. tensor_like is both more informative (it's very easy to say tensor_like of float but otherwise it would be something like float, ndarray of float or TensorVariable of float) and more correct as any type that can be interpreted as an aesara tensor variable is valid.

Reply:
Feel free to make a 2nd review, some will benefit from it because they were merged before we wrote the tensor_like definition for example and don't use it. But I think many (or even most) are already correct (if only the logp or logcdf need changes consider them correct)

Reply:
> Hi @OriolAbril, should we replace `TensorVariable` in the description with :class:`aesara.tensor.TensorVariable`

what about this ?

Reply:
> what about this ?

I think we should use "Aesara tensor" since all other functions have logp the description like this only.

```
value : tensor_like of float
            Value(s) for which log CDF is calculated. If the log CDF for multiple
            values are desired the values must be provided in a numpy array or Aesara tensor.
```

Reply:
Aesara tensor sounds good